### PR TITLE
Add pwd

### PR DESCRIPTION
**Pull request by tiago4orion**
So simple... do makes sense testing this?
I made a test, but feels like I'm testing `os.Getwd` and `fmt.Println` from Go.. there's no much code here.
Makes sense?

Closes #244

### CODE REVIEW

**Comment by tiago4orion** [comment]
oops, the only code here is: https://github.com/NeowayLabs/nash/pull/244/files#diff-7e0bdb2b11a48277e00f1e8c58446148
the others are from #240 that I merged here also

File: Makefile [hunk 1]
```diff
@@ -7,8 +7,9 @@ buildargs = -ldflags "-X main.VersionString=$(version)" -v
 all: build test install
 
 build:
-	go build $(buildargs) -o ./cmd/nash/nash ./cmd/nash
-	go build $(buildargs) -o ./cmd/nashfmt/nashfmt ./cmd/nashfmt
+	cd cmd/nash && go build $(buildargs) 
+	cd cmd/nashfmt && go build $(buildargs) 
+	cd stdbin/mkdir && go build $(buildargs)
```
Comment (vitorarins) [inline Makefile]: This is going to be a big list :sweat_smile:

**Comment by vitorarins** [comment]
@tiago4orion I agree that the code itself is really simple. But does it make sense to have a nash script testing the binary?

**Comment by tiago4orion** [comment]
@vitorarins The code is obviously right. 

_'avoiding bugs by avoiding code' seems to be the only true fact of computer science_

**Pull request merged**